tests/integration/plugins/test_rust_plugin.py

```diff
@@ -0,0 +1,215 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
can you rename this file to test_rust.py so it follows the file naming pattern in that test dir?